scripts/apps/search/components/fields/state.tsx

```diff
@@ -15,7 +15,7 @@ export const state: React.StatelessComponent<any> = (props) => {
             const scheduled = get(props.item, 'archive_item.schedule_settings.utc_publish_schedule');
 
             if (scheduled) {
-                title = gettextCatalog.getString('Scheduled for') + ' ' + datetime.longFormat(scheduled);
+                title = gettext('Scheduled for') + ' ' + datetime.longFormat(scheduled);
```

Comment:
Use interpolation for the entire phrase.

Reply:
Done